.mvn/jvm.config

```diff
@@ -0,0 +1 @@
+--add-opens=java.base/java.lang=ALL-UNNAMED --add-opens=java.base/java.net=ALL-UNNAMED
```

Comment:
if you could get rid of that eye sore end of line.

Reply:
haha, done 🖖